module/VuFind/src/VuFind/Connection/LibGuides.php

```diff
@@ -122,14 +122,57 @@ public function __construct(
      * if an error occurs
      */
     public function getAccounts()
+    {
+        if (!$this->authenticateAndSetHeaders()) {
+            return null;
+        }
+
+        $result = $this->doGet(
+            $this->baseUrl . '/accounts?expand=profile,subjects'
+        );
+
+        if (isset($result->errorCode)) {
+            return null;
+        }
+        return $result;
+    }
+
+    /**
+     * Load all LibGuides AZ databases.
+     *
+     * @return object|null A JSON object of all LibGuides databases, or null
+     * if an error occurs
+     */
+    public function getAZ()
+    {
+        if (!$this->authenticateAndSetHeaders()) {
+            return null;
+        }
+
+        $result = $this->doGet(
+            $this->baseUrl . '/az'
+        );
+
+        if (isset($result->errorCode)) {
+            return null;
+        }
+        return $result;
+    }
+
+    /**
+     * Authentiate to the LibGuides API and set authentiation headers.
+     *
+     * @return bool Indicates if authentiation succeeded.
```

Comment:
Typo: authentiation

Reply:
Somehow I managed to make this same typo three times in the same method doc!!!  Fixed.